src/app/clusters/temperature-control-server/temperature-control-server.cpp

```diff
@@ -134,14 +133,14 @@ bool emberAfTemperatureControlClusterSetTemperatureCallback(app::CommandHandler
             emberAfStatus          = MinTemperature::Get(endpoint, &minTemperature);
             if (emberAfStatus != EMBER_ZCL_STATUS_SUCCESS)
             {
-                status = app::ToInteractionModelStatus(emberAfStatus);
+                status = emberAfStatus;
```

Comment:
Can't this be simplified now that both are the same? This assignement are redendunt now with your last PR, no?

Reply:
+1, you should be able to set status at the start and then change all the if statements to `VerifyOrExit(status == Status::Success)`

Reply:
This is beyond the scope of this PR. Goal is to remove completely the `EmberAfStatus` from the `src/app/util` this optimization should be done in PR #32055 not this one.